src/here-xyz.ts

```diff
@@ -305,17 +289,17 @@ async function execInternalGzip(
     return response;
 }
 
-async function execute(uri: string, method: string, contentType: string, data: any, token: string | null = null, gzip: boolean = false) {
+async function execute(uri: string, method: string, contentType: string, data: any, token: string | null = null, gzip: boolean = false, setAuthorization: boolean = true) {
     if (!token) {
         token = await common.verify();
     }
-    return await execInternal(uri, method, contentType, data, token, gzip);
+    return await execInternal(uri, method, contentType, data, token, gzip, setAuthorization);
 }
 
 program
     .command("list")
     .alias("ls")
-    .description("information about available XYZ spaces")
+    .description("information about available XYZ spaces and projects")
```

Comment:
please revert it back as we are not showing projects in this command

Reply:
Have reverted back projects from list command, Using ```here studio list``` instead